### PR TITLE
Add registration example to respec 

### PR DESCRIPTION
**Pull request by OR13**
Add example to respec

***
<a href="https://pr-preview.s3.amazonaws.com/OR13/vc-specs-dir/pull/10.html" title="Last updated on Apr 4, 2023, 2:34 PM UTC (f356b26)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-specs-dir/10/59dd502...OR13:f356b26.html" title="Last updated on Apr 4, 2023, 2:34 PM UTC (f356b26)">Diff</a>

### CODE REVIEW

**Review by msporny — APPROVED** [review]

**Comment by David-Chadwick** [comment]
I should like to request an addition to this PR, namely to add definitions for each of the properties in the example. The current text is either not there, too brief or ambiguous.

**Comment by OR13** [comment]
@msporny I will add the definitions to this PR, hold on merging.

**Review by jandrieu — APPROVED** [review]
Looks great. Thanks, Orie.

File: index.html [hunk 1]
```diff
@@ -142,18 +142,43 @@ <h1>Adding a Specification Entry</h1>
...
+  "specification": "https://example.github.io/example-spec/",
+  // categories include: vc, credentialStatus, credentialSchema,
+  //                     refreshService, termsOfUse, evidence, and proof
+  "category": "vc",
+  "maintainerEmail": "maintainer@community.example",
+  // These fields are optional
+  "maintainerName": "Example Community Group",
+  "maintainerWebsite": "https://example.github.io/",
+  // RDF vocabularies in yml2vocab format
+  "vocabulary": ["https://example.github.io/vocabularies/example.yml"]
```
Comment (msporny) [inline index.html:170]: Should we have comments in the example? I get that it helps explain the example; it's also invalid JSON.

I'm fine either way, just raising it as a concern.

Comment (OR13) [reply index.html]: I removed the comments

**Review by msporny — APPROVED** [review]

**Pull request merged**